### ISSUE TITLE
How to use with libpurple

### ISSUE THREAD

**Issue by Joeordie** [jgeboski/bitlbee-facebook#37]
I am having a great deal of trouble getting this to make on archlinux (armv7h) with bitlbee 3.4.1 compiled with libpurple.  

Have you tried connecting the plugin after compilation to use libpurple?

**Comment by jgeboski** [comment]
I think you want: https://github.com/jgeboski/purple-facebook/wiki

**Comment by Joeordie** [comment]
Actually im not so sure, 

I am very new to compiling my own stuff.  So keep in mind i may be making a very basic error what i was shooting for, which maybe beyond this scope was to create a bitlbee installation with: 

MQTT facebook (plugin)
telegram  (libpurple using telegram-purple) https://github.com/majn/telegram-purple

I have successfully compiled libpurple (telegram-purple) and then rolled bitlbee in with telegram support, but whenever i try to add on this plugin bitlbee-facebook i throw errors.

I have tried this several ways in the attempt of being slick.  For example, compile the plugin with bitlbee-3.4.1 depends from the repo, install, then recompile bitlbee with telegram-purple support.  

Tonight i will duplicate steps and get errors to post. 

My memory tells me that there are problems with the "CFLAGS" or the "LIBS"   even when i set them directly (which i could be getting wrong)  

Its a learning evolution, but I feel like telegram and facebook support for bitlbee isn't an insurmountable request. 

If i can navigate this task ill type up and add to any wiki's of your choosing.
